pkgs/by-name/de/devcontainer/package.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchYarnDeps
+, fixup_yarn_lock
```

Comment:
:ou are picking the `fixup-yarn-lock` from somewhere else, so this is not needed.
```suggestion
```